crates/hyperqueue/src/client/commands/worker.rs

```diff
@@ -20,6 +20,32 @@ use tako::worker::rpc::run_worker;
 use tokio::net::lookup_host;
 use tokio::task::LocalSet;
 
+
+#[derive(Debug)]
+pub enum WorkerFilter {
+    All,
```

Comment:
Let's remove the `All` variant, it's not needed and it doesn't make sense to use e.g. `--filter all,running`. If the filter set will be empty, all workers will be returned.